pipelines/main/platforms/build_macos.yml

```diff
@@ -32,7 +32,7 @@ steps:
           ./julia -e 'using InteractiveUtils; InteractiveUtils.versioninfo()'
 
           echo "--- Create build artifacts"
-          make --output-sync -j 8 binary-dist ${MAKE_FLAGS?}
+          gmake --output-sync -j 8 binary-dist ${MAKE_FLAGS?}
```

Comment:
I don't think you want `gmake`.  This isn't FreeBSD, Mac's have GNU Make by default:

```
$ make --version
GNU Make 3.81
```

Reply:
I tried that at first https://buildkite.com/julialang/julia-buildkite/builds/112#9229a63f-d883-4f03-8ccc-08ef8e751148 but `--output-sync` is only available on newer make. Make 3.81 is from 2006 😄 .

Reply:
I guess I can just make without it, it's not that big of an issue. 
It seems like aarch64 has it and x86 doesn't for some reason.